dev-tools/omdb/tests/successes.out

```diff
@@ -589,8 +589,9 @@ task: "region_replacement"
   currently executing: no
   last completed activation: <REDACTED ITERATIONS>, triggered by a periodic timer firing
     started at <REDACTED_TIMESTAMP> (<REDACTED DURATION>s ago) and ran for <REDACTED DURATION>ms
-    number of region replacements started ok: 0
-    number of region replacement start errors: 0
+    number of region replacement requests created ok: 0
+    number of region replacement start sagas started ok: 0
+    number of errors: 0
```

Comment:
take it or leave it: if you want to be super-duper extra fancy and align the number columns programmatically, you *could* use the approach I took in #6541, using `const_max_len` to calculate the longest line to align to: https://github.com/oxidecomputer/omicron/blob/a3917ea5df727460392da3734db403fd0997d742/dev-tools/omdb/src/bin/omdb/nexus.rs#L1193-L1225

totally up to you, though --- this is probably just me being a perfectionist.

Reply:
in this case, I'm going to leave it the way it is